src/js/front-end/removeDivider.js

```diff
@@ -0,0 +1,20 @@
+const diCloumns=document.querySelectorAll(".has-dividers")
+
+if(diCloumns&&diCloumns.length>0){
+    const width = window.innerWidth-48;
```

Comment:
I would suggest instead of just making this value `48` to actually get the padding property of the highest level section, that way if we ever change it in the css we don't need to remember to come back here and change this number.

Reply:
I agree not to use a fixed value, and also the columns could be in some containers. Maybe instead of using the window width, use the width of the parent of the columns instead?

Reply:
In that case we should add all the paddings together?

Reply:
That's a good idea too, except what if the parent of `.columns` has padding, then the widths wouldn't be equal. So you'd have to make sure to check for padding on the parent and if there is padding then add that to the width of `.columns`.